submissions/TatyanaMolchanova/HTML-CSS-Popup/styles.css

```diff
@@ -0,0 +1,117 @@
+.nav-link img {
+    max-width: 30px;
+}
+.nav {
```

Comment:
Use flexbox of CSS grid for creating layout they are more appropriate technologies for that. Float not a good choice for it.)

Reply:
`.nav {
    display: flex;
    align-items: flex-end;
    flex-direction: column;
}`

I changes float: right to display:flex